examples/integer-ranges/integer_ranges.bal

```diff
@@ -1,21 +1,23 @@
 import ballerina/io;
 
 public function main() {
-    // A closed integer range `x ... y` generates an array of integers containing values `x` to `y`, `x` and `y`
-    // inclusive
+    // A closed integer range `x ... y` generates an array of integers
+    // containing values `x` to `y`, including `x` and `y`.
     int[] closedRangeArray = 25 ... 28;
+    int closedArrayLength = closedRangeArray.length();
```

Comment:
IMO `closedArrayLength` and `halfOpenArrayLength` sound like closed and half open are properties related to the arrays.

Shall we just say `arrayLength` and use the same variable below too?